sql/updates/latest-dev.sql

```diff
@@ -1,2 +1,21 @@
 DROP FUNCTION IF EXISTS recompress_chunk;
 
+
+-- See the comments for ContinuousAggsBucketFunction structure.
+CREATE TABLE IF NOT EXISTS _timescaledb_catalog.continuous_aggs_bucket_function(
+  mat_hypertable_id integer PRIMARY KEY REFERENCES _timescaledb_catalog.hypertable (id) ON DELETE CASCADE,
+  experimental bool NOT NULL,
+  name text NOT NULL,
```

Comment:
```suggestion
  name name NOT NULL,
```

Reply:
I'm marking this as resolved, see https://github.com/timescale/timescaledb/pull/3753#issuecomment-991073467